transformers/file_utils.py

```diff
@@ -120,6 +120,10 @@ def url_to_filename(url, etag=None):
         etag_hash = sha256(etag_bytes)
         filename += '.' + etag_hash.hexdigest()
 
+    if xla_device:
+        import torch_xla.core.xla_model as xm
+        filename += '.xla' + str(xm.get_ordinal())
```

Comment:
So we really are ok with save / load multiple copies then?

Reply:
Yeah I believe so. That was what Davide suggested we should do.

Reply:
let's revisit that later, since these models can be pretty big. fine for now.

Reply:
What we'd need are some synchronization primitives but the current upstream PT doesn't play well with Python native `multiprocessing` objects: https://github.com/pytorch/xla/issues/1354.